sdk/batch/batch/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Release History
+
+## 10.1.0 (2022-02-09)
+
+### Features
+
+- Added property `UploadHeaders` to `OutputFileBlobContainerDestination`.
```

Comment:
Should UploadHeaders be uploadHeaders here?

Reply:
Yes good catch! Will make the change!